application/sitebuilder/build.py

```diff
@@ -62,6 +76,35 @@ def enforce_static_mode():
             clear_up(build_dir)
 
 
+def build_and_upload_error_pages(application):
+    with application.app_context():
+        build_dir = new_build_dir(application)
+
+        # Inject static_mode=True into all Jinja render_template calls so that pages are automatically rendered in the
+        # correct mode.
+        @application.context_processor
+        def enforce_static_mode():
+            return dict(static_mode=True)
+
+        local_build = application.config["LOCAL_BUILD"]
+
+        build_error_pages(build_dir)
+
+        print("Deploy site (error pages) to S3: ", application.config["DEPLOY_SITE"])
+        if application.config["DEPLOY_SITE"]:
+            from application.sitebuilder.build_service import s3_deployer, _upload_dir_to_s3
+            from application.cms.file_service import S3FileSystem
+
+            s3 = S3FileSystem(
+                application.config["S3_STATIC_SITE_ERROR_PAGES_BUCKET"], region=application.config["S3_REGION"]
```

Comment:
I thought S3 buckets in AWS were unique across all regions, so maybe we don't need to specify it here? (The region doesn't seem to be part of the S3 ARN identifier?)

Reply:
This is the normal/existing interface to our S3FileSystem class, so it's not really something worth changing for this?

Reply:
Probably not. Although now we share the region name across two different buckets, which is maybe unexpected (not that there's any good reason to have them in different regions).

Reply:
🙈